inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -595,18 +612,28 @@ void MKLDNNROIPoolingNode::execute() {
                 float height_scale = (jpp.pooled_h > 1 ? ((roi_end_h_ - roi_start_h_) * (jpp.ih - 1)) / (jpp.pooled_h - 1) : 0);
```

Comment:
We have comment in the test but have no explanation in source code.
Could you please leave a comment about handling corner case ("why" explanation) in the key sections

Reply:
Done